index.css

```diff
@@ -0,0 +1,1988 @@
+/* Set the defaults */
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  overflow-x: hidden;
+  max-width: 90em;
+  margin: 0 auto;
+  font-family: "ARS Maquette", "Helvetica Neue", Arial, sans-serif;
+  font-size: 100%;
+  font-weight: 400;
+  line-height: 1.125em;
+  height: 100%;
+  padding: 0;
+}
```

Comment:
- [ ] Your body doesn't have a 100% width like the original page. Please make this change.